test/integration/test_cli_commands.py

```diff
@@ -874,3 +874,20 @@ def test_logging_with_logformat_logstash_flag(self):
                     obj={"filters":[]})
         output = sorted(result.output.splitlines(), reverse=True)[:4]
         self.assertEqual(expected, output)
+
+class TestCLIRepositoryCreate(CuratorTestCase):
+    def test_create_fs_repository(self):
+        test = clicktest.CliRunner()
+        result = test.invoke(
+                    curator.repomgrcli,
+                    [
+                        '--logfile', os.devnull,
+                        '--host', host,
```

Comment:
Testing this against `localhost` does not really tell if the erroneous behavior I experience is fixed. It'd be nice to be able to test against _any_ host, even if the connection fails.